Formula/php.rb

```diff
@@ -0,0 +1,343 @@
+      --enable-ftp
+      --enable-fpm
+      --enable-intl
+      --enable-mbregex
+      --enable-mbstring
+      --enable-mysqlnd
+      --enable-opcache-file
+      --enable-pcntl
+      --enable-phpdbg
+      --enable-phpdbg-webhelper
+      --enable-shmop
+      --enable-soap
+      --enable-sockets
+      --enable-sysvmsg
+      --enable-sysvsem
+      --enable-sysvshm
+      --enable-wddx
+      --enable-zip
+      --libexecdir=#{libexec}
+      --with-apxs2=#{Formula["httpd"].opt_bin}/apxs
+      --with-bz2
+      --with-fpm-user=_www
+      --with-fpm-group=_www
+      --with-freetype-dir=#{Formula["freetype"].opt_prefix}
+      --with-gd
+      --with-gettext=#{Formula["gettext"].opt_prefix}
+      --with-gmp=#{Formula["gmp"].opt_prefix}
+      --with-icu-dir=#{Formula["icu4c"].opt_prefix}
+      --with-imap=shared,#{Formula["imap-uw"].opt_prefix}
+      --with-imap-ssl=#{Formula["openssl"].opt_prefix}
+      --with-jpeg-dir=#{Formula["jpeg"].opt_prefix}
+      --with-kerberos
+      --with-ldap=shared
+      --with-ldap-sasl
+      --with-libedit
+      --with-libzip
+      --with-mhash
+      --with-mysql-sock=/tmp/mysql.sock
+      --with-mysqli=mysqlnd
+      --with-ndbm
+      --with-openssl=#{Formula["openssl"].opt_prefix}
+      --with-password-argon2=#{Formula["argon2"].opt_prefix}
+      --with-pdo-mysql=mysqlnd
+      --with-pdo-odbc=unixODBC,#{Formula["unixodbc"].opt_prefix}
+      --with-pdo-pgsql=#{Formula["libpq"].opt_prefix}
+      --with-pgsql=#{Formula["libpq"].opt_prefix}
+      --with-pic
+      --with-png-dir=#{Formula["libpng"].opt_prefix}
+      --with-pspell=#{Formula["aspell"].opt_prefix}
+      --with-snmp=shared
+      --with-sodium=#{Formula["libsodium"].opt_prefix}
+      --with-tidy
+      --with-unixODBC=#{Formula["unixodbc"].opt_prefix}
+      --with-webp-dir=#{Formula["webp"].opt_prefix}
+      --with-xmlrpc
+      --with-xsl
+      --with-zlib
+    ]
+
+    if MacOS.version < :lion
+      args << "--with-curl=#{Formula["curl"].opt_prefix}"
+    else
+      args << "--with-curl"
+    end
+
+    system "./configure", *args
+
+    inreplace "Makefile" do |s|
+      # Custom location for php module and remove -a (don't touch httpd.conf)
+      s.gsub! /^INSTALL_IT = \$\(mkinstalldirs\) '([^']+)' (.+) LIBEXECDIR=([^\s]+) (.+) -a (.+)$/,
+        "INSTALL_IT = $(mkinstalldirs) '#{lib}/httpd/modules' \\2 LIBEXECDIR='#{lib}/httpd/modules' \\4 \\5"
+
+      # Reorder linker flags to put system paths at the end to avoid accidential system linkage
+      %w[EXTRA_LDFLAGS EXTRA_LDFLAGS_PROGRAM].each do |mk_var|
+        system_libs = []
+        other_flags = []
+        s.get_make_var(mk_var).split.each do |f|
+          if f[%r{^-L\/(?:Applications|usr\/lib)\/}]
+            system_libs << f
+            next
+          end
+          other_flags << f
+        end
+        s.change_make_var! mk_var, other_flags.concat(system_libs).join(" ")
+      end
+    end
+
+    system "make"
+    ENV.deparallelize
```

Comment:
Parallelized `make install` fails? Has that been reported?

Reply:
https://externals.io/message/100838#100853 recent attempt to fix it, doesn't seem to be going anywhere.

Reply:
Ignore that, I can't read apparently

Reply:
I don't think it fails with any sort of regularity or predictability. I took it out in my fork and things work well. Occasionally, I've run into issues where the pear binary doesn't get installed, but we could write a test to catch that (maybe unrelated).